Realm/RLMSyncSession.h

```diff
@@ -56,6 +105,26 @@ NS_ASSUME_NONNULL_BEGIN
 /// session.
 - (nullable RLMSyncConfiguration *)configuration;
 
-NS_ASSUME_NONNULL_END
+/**
+ Register a progress notification block. Multiple blocks can be registered with the same
```

Comment:
The first sentence should probably stand alone as the one-line summary as the method, with the remainder of the paragraph being moved to a second paragraph.